src/codegen/sys/cargo_toml.rs

```diff
@@ -82,7 +82,7 @@ fn fill_in(root: &mut Table, env: &Env) {
 
     {
         let build_deps = upsert_table(root, "build-dependencies");
-        set_string(build_deps, "pkg-config", "0.3.7");
+        set_string(build_deps, "system-deps", "1.3");
```

Comment:
Maybe add `unset(build_deps,  "pkg-config");` as it no more needed

Reply:
Good point, I added that.